### PR TITLE
This commit contains following changes:

### PR DESCRIPTION
**Pull request by agarwal-kritika**
1.Update PG version to 17rc1
2.Update LP version to 5.0
3.Enable PG docs build
4.Update python version to 3.12.5, perl to 5.40.0 and tcl is 8.6.14 5.Update pgAdmin script: build-pgadmin.sh.in according to community pgAdmin

### CODE REVIEW

File: server/build-osx.sh [hunk 1]
```diff
@@ -270,7 +270,7 @@ _build_server_osx() {
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libtiff*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libtiff"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libiconv*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libiconv"
     ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libpng16*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libpng16"
-    #ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libexpat*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libexpat"
+    ssh $PG_SSH_OSX "cp -pR /opt/local/Current_v15/lib/libexpat*.dylib $PG_STAGING/stackbuilder.app/Contents/Frameworks/" || _die "Failed to copy the latest libexpat"
```
Comment (sandeep-edb) [inline server/build-osx.sh:273]: why is expat needed now? does the license file include it's license?

File: settings.sh.in [hunk 2]
```diff
@@ -70,38 +70,34 @@ PGADMIN_KRB5_DIR="C:\\\\Program Files\\\\MIT\\\\Kerberos"
...
 
 # CFLAGS/CPPFLAGS/CXXFLAGS settings for different platforms. The is most important for Mac
 # where we may need to use non-default SDKs if we're building on a platform newer than the
 # oldest supported
 
-MACOSX_MIN_VERSION=10.15
+MACOSX_MIN_VERSION=12
 ARCHFLAGS="-arch x86_64 -arch arm64"
-SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk
+SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```
Comment (sandeep-edb) [inline settings.sh.in]: use SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk

so that SDK version can be known

File: versions.sh [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 # Source tarball versions
 
-PG_TARBALL_POSTGRESQL=17beta3
-PG_TARBALL_PGADMIN=8.10
-PG_LP_VERSION=4.3
+PG_TARBALL_POSTGRESQL=17rc1
+PG_TARBALL_PGADMIN=8.11
+PG_LP_VERSION=5.0
```
Comment (sandeep-edb) [inline versions.sh:7]: shouldn't it be rc1 installer?

File: settings.sh.in (hunk 2 above)
Comment (agarwal-kritika) [reply settings.sh.in]: This is the softlink which was created when i installed this SDK, it always points to the latest.

File: versions.sh (hunk 3 above)
Comment (agarwal-kritika) [reply versions.sh:7]: Right now the LP installers is created with the name: edb-languagepack-5.0-1-osx.dmg, what should be the name of the installer then?

File: server/build-osx.sh (hunk 1 above)
Comment (agarwal-kritika) [reply server/build-osx.sh:273]: While testing the installer, SB was not launching if i dont bundle expat, it was throwing error for expat. So added it back to the script. Yes license file has its license.

File: settings.sh.in (hunk 2 above)
Comment (sandeep-edb) [reply settings.sh.in]: we want latest of 14, so MacOSX14.sdk is what we need

**Review by sandeep-edb — APPROVED** [review]
LGTM

**Pull request merged**